__tests__/articles.tests.js

```diff
@@ -13,6 +13,36 @@ afterAll(() => {
 })
 
 describe('/api/articles', () => {
+    it('should return all articles', async () => {
+        const { body } = await request(app)
+        .get('/api/articles')
+        .expect(200)
+        body.forEach((article) => {
+            expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(Number)
+            })
+        })
+    });
```

Comment:
Two things you can add to the above test - 
- when you assert inside a forEach in a test, if the array of articles is empty then your forEach will give you a false positive. To guard against this you can add a length check on the array before you do the forEach. How many articles are you expecting back? Check the test data 
- Looks like you have ordered the responses in the model by created_at DESC, you should test for that here. jest-sorted could be useful, there's info in the notes and on the jest-sorted npm page if you haven't used it before!